src/questions/BaseQuestion.js

```diff
@@ -0,0 +1,17 @@
+export default class BaseQuestion {
+    constructor (data) {
+        this.data = data;
+        this.answers = data.answers;
+        this.state = data.state;
+        this.answer = data.answer;
+        this.packages = data.packages;
+        this.devPackages = data.devPackages;
+    }
+
+    retrievePackages = () => {
```

Comment:
Vrij onnodige functie. Lijkt me logischer als functies gewoon direct `this.packages` etc gebruiken ipv een extra functie.